pkgs/development/python-modules/inflate64/default.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+
+, setuptools
+, setuptools-scm
+}:
+
+buildPythonPackage rec {
+  pname = "inflate64";
+  version = "1.0.0";
+
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+
+    hash = "sha256-MniCe4A88Aah3yUfPhM3TH0m23eeWjMynMEXibgEvC0=";
+  };
+  build-system = [
+    setuptools
+    setuptools-scm
+  ];
+
+  pythonImportsCheck = [ "inflate64" ];
+
+  meta = with lib; {
```

```suggestion
  meta = {
```
tracking issue: #292468